package.json

```diff
@@ -11,30 +11,30 @@
     "format": "prettier --write ."
   },
   "devDependencies": {
-    "turbo": "1.10.7",
-    "typescript": "5.1.6",
-    "ts-node": "10.9.1",
-    "@tsconfig/node18": "1.0.1",
-    "@types/node": "20.5.3",
-    "@babel/core": "7.22.10",
-    "@babel/preset-env": "7.22.10",
-    "@babel/preset-typescript": "7.22.5",
-    "@types/jest": "29.5.4",
-    "jest": "29.6.3",
+    "@babel/core": "7.22.17",
+    "@babel/preset-env": "7.22.15",
+    "@babel/preset-typescript": "7.22.15",
     "@jest/globals": "29.6.4",
+    "@tsconfig/node18": "^18.2.0",
+    "@types/jest": "29.5.4",
+    "@types/node": "^18",
     "@types/supertest": "2.0.12",
-    "supertest": "6.3.3",
-    "graphql": "^16.0.0 || ^17.0.0",
-    "eslint-config-standard-with-typescript": "39.0.0",
-    "@typescript-eslint/eslint-plugin": "6.4.0",
-    "eslint": "8.0.1",
-    "prettier": "3.0.3",
-    "eslint-plugin-import": "2.25.2",
-    "eslint-plugin-n": "15.0.0",
-    "eslint-plugin-promise": "6.0.0",
+    "@typescript-eslint/eslint-plugin": "6.6.0",
+    "eslint": "8.48.0",
     "eslint-config-prettier": "9.0.0",
-    "eslint-plugin-prettier": "5.0.0",
-    "eslint-plugin-react-hooks": "4.3.0"
+    "eslint-config-standard-with-typescript": "39.0.0",
+    "eslint-plugin-import": "2.28.1",
+    "eslint-plugin-n": "^16.0.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-promise": "^6.1.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
I am curious to hear your thoughts @Tohaker on pinning vs. nonpinning dependencies in this project. 

Should we pin everything? Should we pin some and not others?

Reply:
With the addition of https://github.com/rocket-connect/graphql-debugger/issues/23 could we pin everything and then rely on that to apply patches etc.?

Reply:
I pin `dependencies` and leave `devDependencies` unpinned. That way we control what's released to users but allow developers to have the patched versions of what they need to work with.

However, once we have an automated process bumping dependency versions whenever they're available, I'm comfortable to have _everything_ pinned.

Reply:
I like the idea of pinning the main ones and keeping the dev ones floating. 

Also, something to note here is that we have one expectation in the mono repo and that 

```
peerDependencies
    "graphql": "^16.0.0 || ^17.0.0"
```

In the utils package. 

Keen to hear thoughts on if this is the best approach to incude GraphQL in your project without clashing with other versions: 

```
Cannot use GraphQLSchema "[object Object]" from another module or realm.
```

Reply:
I think that _should_ work in theory, although when I tried adding this one locally to my own existing (albeit far out of date) project it threw the error above. Might need to triage whether that's a setup issue we have or if it was just my crummy code 😆